packages/backend/src/graphql/queries/test-connection.ts

```diff
@@ -52,8 +53,15 @@ const testConnection = async (
   let isStillVerified
   try {
     isStillVerified = !!(await app.auth.isStillVerified($))
-  } catch {
+  } catch (err) {
     isStillVerified = false
+    logger.error(`Error verifying CONNECTION ID: ${params.connectionId}`, {
+      errorDetails: {
+        stepId: params.stepId,
```

Comment:
Any reason we're nesting this inside `errorDetails`? Typically stepId, etc etc are at the root level of a log - e.g. https://github.com/opengovsg/plumber/blob/cd8ccd11d77841cdb215581b03dd2dd4870be68c/packages/backend/src/helpers/backoff.ts#L48

aside: at some point we need to standardize how we log and add lints, otherwise its going to be harder and harder to search for stuff in DD